frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -879,7 +879,7 @@ class CreateInterface extends React.Component {
                             </OverlayTrigger>
                           )}
 
-                          {this.state.task.cur_round > 1 && (
+                          {/* {this.state.task.cur_round > 1 && (
```

Comment:
is there a reason why it's commented instead of deleted? I think deleting should be fine - if this feature is to be reinstated for some reason at some point in the future the devs can come back to this commit anyways

Reply:
Yeah I had the same doubt, so I left it commented. Maybe @TristanThrush can double confirm this, then I would remove.